Clarify nightly style policy (#180)
rust-lang/style-team: nightly-style-procedure.md

The style guide may or may not prescribe formatting behavior for nightly syntax

> in preparation for a feature's eventual stabilization, and to record in
> principal the behavior implemented by rustfmt around nightly syntax, but that

Author: ```suggestion
in preparation for a feature's eventual stabilization or to reflect actual
the behavior implemented by rustfmt around nightly syntax, but that
```

> The style guide may or may not prescribe formatting behavior for nightly syntax
> in preparation for a feature's eventual stabilization or to reflect actual
> the behavior implemented by rustfmt around nightly syntax, but that
> behavior is not binding until the feature is stabilized.

Author: ```suggestion
The style guide may or may not prescribe formatting behavior for nightly syntax
in the "nightly" chapterin preparation for a feature's eventual stabilization
or to reflect actual the behavior implemented by rustfmt around nightly syntax,
but that behavior is not binding until the feature is stabilized and the text is moved
into the appropriate section of the style guide.
```
